documentation/API_documentation/telco-index-User-Story.md

```diff
@@ -0,0 +1,24 @@
+**User Story: Credit scoring service for banking**
+<br>
+
+| **Item** | **Details** |
+| -------- | ----------- |
+| ***Summary*** | As a bank officer, I want to evaluate a micro-credit application for a customer who has no credit history within our bank. I use the Customer Insights API to gather additional information about the customer's telco behavior to assess their creditworthiness. |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User<br> **Actors:** Bank Officer, Bank System, Communication Service Provider (CSP)<br> **Scope:** Retrieve a relevant score calculated from the data of the CSP, enabling the bank to make an informed decision on micro-credit approval. |
+| ***Pre-conditions*** | The preconditions are listed below:<br><ol><li>The bank is integrated with the CSP's Customer Insights API.</li><li>The customer has consented to sharing their telco data for credit assessment purposes.</li><li>The bank officer has the necessary credentials to access the Customer Insights API.</li></ol> |
+| ***Activities/Steps*** | **Begins when:** The bank officer submits a micro-credit application for a customer without an existing credit history.<br><br>**Steps include:**<br><ol><li>The bank system makes a request to the Customer Insights API using customer consented information.</li><li>The CSP processes the request and gathers telco-related behavioral insights.</li><li>The CSP sends a calculated score to the bank system.</li><li>The bank officer evaluates the score to assess the credit risk of the customer.</li></ol><br> **Ends when:** The bank officer either approves or declines the micro-credit application based on the insights provided. |
+| ***Post-conditions*** | The bank officer has sufficient data to approve or deny the micro-credit request for a customer without a traditional credit history. |
+| ***Exceptions*** | Several exceptions might occur during the Customer Insights API operations:<br>- Unauthorized: Invalid credentials provided by the bank system.<br>- Consent missing: The customer has not provided consent for sharing telco data (i.e. permission denied).<br>- Data not found: Insufficient user data available for the customer.<br>- Corporate customers: The API is not applicable for corporate customers. |
+
+
+**User Story: Credit scoring service for riders**
```

Comment:
There are multiple APIs and multiple service providers in this story - this may be confusing.